concourse/pipelines/create-bosh-cloudfoundry.yml

```diff
@@ -1222,15 +1238,82 @@ jobs:
               export CONCOURSE_ATC_PASSWORD
               ./paas-cf/concourse/scripts/bosh-cli.sh bosh status
 
-
-  - name: tag-release
+  - name: performance-tests
     plan:
       - aggregate:
           - get: pipeline-trigger
             passed: ['post-deploy', 'smoke-tests', 'acceptance-tests', 'custom-acceptance-tests', 'bosh-cli-test']
             trigger: true
           - get: paas-cf
             passed: ['post-deploy', 'smoke-tests', 'acceptance-tests', 'custom-acceptance-tests', 'bosh-cli-test']
+          - get: cf-manifest
+            passed: ['post-deploy', 'smoke-tests', 'acceptance-tests', 'custom-acceptance-tests']
+          - get: bosh-CA
+      - task: generate-test-config
+        config:
+          platform: linux
+          image: docker:///ruby#2.2-slim
+          inputs:
+            - name: paas-cf
+            - name: cf-manifest
+          outputs:
+            - name: test-config
+          run:
+            path: sh
+            args:
+              - -e
+              - -c
+              - |
+                export CF_MANIFEST=cf-manifest/cf-manifest.yml
+                ./paas-cf/tests/performance-tests/generate_test_config.rb \
+                  > test-config/config.json
+
```

Comment:
> Yes, that was the idea. To keep it for analysis so we are able to graph historical data.

I'm unsure about storing it in an arbitrary format while we don't have a means to access/process it. We could store in Graphite, but I think it's OK not to store it anywhere for this spike.